zerver/openapi/zulip.yaml

```diff
@@ -307,6 +315,51 @@ paths:
                                   "setting": true,
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event sent to a user's clients when that user's settings
+                                have changed.
+
+                                **Changes**: New in Zulip 5.0 (feature level 87), replacing the
+                                previous `update_display_settings` event type, which is still
```

Comment:
This should mention notification settings too, right?

Reply:
Yeah missed that, updated now.